graph-db/serverless/src/IndexCollectionAcl/handler.js

```diff
@@ -0,0 +1,66 @@
+import 'array-foreach-async'
+import gremlin from 'gremlin'
+
+import { getEchoToken } from '../utils/cmr/getEchoToken'
+import { initializeGremlinConnection } from '../utils/gremlin/initializeGremlinConnection'
+import { indexCollectionAcl } from '../utils/cmr/indexCollectionAcl'
+import { deleteCollectionAcl } from '../utils/cmr/deleteCollectionAcl'
+import { fetchAccessControlList } from '../utils/cmr/fetchAccessControlList'
+const gremlinStatistics = gremlin.process.statics //Used for anonymous traversals commonly imported as __
+let gremlinConnection
+let token
+
+const updateActionType = 'concept-update'
+const deleteActionType = 'concept-delete'
+
+const indexAcl = async (event) => {
+    // Prevent creating more tokens than necessary
+  if (token === undefined) {
+      token = await getEchoToken()
+  }
+    // Prevent connecting to Gremlin more than necessary
+  if (!gremlinConnection) {
+      gremlinConnection = initializeGremlinConnection()
+  }
+  const { 'concept-id': conceptId, 'revision-id': revisionId, 'action':action } = event
+  console.log(conceptId)
+  console.log(revisionId)
+  console.log(action)
```

Comment:
Would recommend merging this into a formatted string for easier reading
Somehing like `console.log('action[${action}] on  [${conceptId}][${revisionId}]'`